.circleci/config.yml

```diff
@@ -184,15 +152,15 @@ workflows:
           filters:
             branches:
               only:
-                - '/release-0\.(12|13|14|15|16|23)/'
```

Comment:
Was this an accident that the 23 got removed, or is this intentional?

Reply:
Oh shoot, yeah, I have no idea how that happened. Good catch!

Reply:
The problem was caused by me failing to update the .circleci/config.yml.j2 file when I originally added 23 to the supported list. I committed a change taken from ap-airflow that adds a pre-commit hook to automatically run the generation whenever those files change.